### PR TITLE
Show at least two lemmas for lexicalized dimunitive forms 

### PR DESCRIPTION
**Pull request by Madoshakalaka**
the importing process is fixed

### CODE REVIEW

File: CreeDictionary/DatabaseManager/xml_entry_lemma_finder.py [hunk 1]
```diff
@@ -110,21 +111,40 @@ def extract_fst_lemmas(
 
                 ambiguities: List[Analysis] = []
 
+                # put priority in looking for an unique and identical wordform
```
Comment (eddieantonio) [inline CreeDictionary/DatabaseManager/xml_entry_lemma_finder.py]:
```suggestion
                # put priority in looking for a unique and identical wordform
```

File: package.json [hunk 2]
```diff
@@ -4,10 +4,10 @@
   "description": "Cree Intelligent Dictionary",
   "scripts": {
     "start": "rollup --config --watch & pipenv run dev",
-    "bg-watch": "rollup --watch &",
+    "bg-watch": "rollup -c && rollup --watch &",
```
Comment (eddieantonio) [inline package.json]: Interesting! Why this change?

File: cypress/integration/regressions.spec.js [hunk 3]
```diff
@@ -97,10 +97,46 @@ context('Regressions', () => {
...
+    })
+  })
+  
+  // https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/issues/176
+  describe('should show at least two lemmas for lexicalized diminutive forms', ()=>{
+    it('should show atâhk and acâhkos for acâhkos', () => {
+      cy.visit('/search/acâhkos')
+      cy.get('[data-cy=search-results]')
+        .should('contain', 'atâhk')
+        .and('contain', 'acâhkos')
```
Comment (eddieantonio) [inline cypress/integration/regressions.spec.js]: Nice!

**Comment by Madoshakalaka** [comment]
@eddieantonio Need you to checkout this branch and do a `manage-db import /res/dictionaries/ --multi-process 2` and `push-db` before this is merged because I don't have access to Sapir.

Merging right now won't have any impact in the production environment though, just meaning we need to manually update database later

File: package.json (hunk 2 above)
Comment (Madoshakalaka) [reply package.json]: I think `rollup --watch` does not build anything when you start the command. I may be wrong but adding rollup -c has no impact

File: cypress/integration/regressions.spec.js (hunk 3 above)
Comment (Madoshakalaka) [reply cypress/integration/regressions.spec.js]: yeet!
